tomcat/src/main/java/org/apache/coyote/http11/controller/LoginController.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.coyote.http11.controller;
+
+import nextstep.jwp.model.Request;
+import org.apache.coyote.http11.service.LoginService;
+import nextstep.jwp.vo.FileName;
+import nextstep.jwp.vo.FormData;
+import nextstep.jwp.vo.Response;
+
+import java.io.IOException;
+
+public class LoginController implements Controller {
+
+    private static final LoginController INSTANCE = new LoginController();
+
+    @Override
+    public Response respond(Request request) throws IOException {
+        FileName fileName = request.getFileName();
```

Comment:
```suggestion
    public Response respond(Request request) throws {
```
fileName 변수와 IOException이 사용되지 않고 있어요!

Reply:
수정하였습니다